translations/TranslationServer.js

```diff
@@ -19,8 +19,12 @@ if (typeof hoot === 'undefined') {
     hoot.Settings.set({"ogr.note.extra": "attribute"});
     hoot.Settings.set({"reader.add.source.datetime": "false"});
     hoot.Settings.set({"writer.include.circular.error.tags": "false"});
+    hoot.Settings.set({"ogr.thematic.structure":"false"});
```

Comment:
@mattjdnv This change doesn't really have a practical impact does it?  The output is only ever going to osm format, not an ogr format like shp or gdb.